features/prefers-reduced-transparency.yml

```
@@ -0,0 +1,9 @@
+name: prefers-reduced-transparency media query
+description: The `prefers-reduced-transparency` CSS media query sets styles based on whether the user prefers to reduce the amount of transparent effects on their device, for example to improve contrast and readability.
```

nit, needless pedantry:

```suggestion
description: The `prefers-reduced-transparency` CSS media query sets styles based on whether the user prefers to reduce the amount of transparent effects on their device, for example to improve contrast and legibility.
```